Pal/include/pal/pal.h

```diff
@@ -447,44 +449,49 @@ DkProcessExit(PAL_NUM exitCode);
 
 /*! Stream Access Flags */
 enum PAL_ACCESS {
-    PAL_ACCESS_RDONLY = 00,
-    PAL_ACCESS_WRONLY = 01,
-    PAL_ACCESS_RDWR   = 02,
-    PAL_ACCESS_APPEND = 04,
-    PAL_ACCESS_MASK   = 07,
+    PAL_ACCESS_RDONLY = 0,
+    PAL_ACCESS_WRONLY = 1,
+    PAL_ACCESS_RDWR   = 2,
+    PAL_ACCESS_APPEND = 4,
+    PAL_ACCESS_MASK   = 7,
```

Comment:
Here you removed the leading 0 for octal while for the flags below you kept it. Isn't this a bit inconsistent?